sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -193,12 +193,17 @@ def trace_link_message(events, parent_span=None):
                     if event.properties:
                         traceparent = event.properties.get(b"Diagnostic-Id", "").decode("ascii")
                         if traceparent:
-                            current_span.link(traceparent)
+                            try:
+                                current_span.link(
+                                    traceparent,
+                                    attributes={"enqueuedTime": event.message.annotations.get(PROP_TIMESTAMP)}
```

Comment:
Is there a chance that the annotation in None? And then, do we want to annotate {"enqueuedTime": None"}?

Reply:
Nope. All received events have this system property